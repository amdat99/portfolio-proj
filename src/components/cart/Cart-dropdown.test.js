import React from 'react';

import { shallow } from 'enzyme';
import {createSerializer} from 'enzyme-to-json';
import toJson from 'enzyme-to-json';

import CartDropdown from './Cart-dropdown'

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

it('expect to render Greeting component', () => {
    const mockCartItems = [ {
        name: 'ball',
        price: '5',
        seller: 'john',
        image: 'picture',
        id: 'id'
    }]
    expect(toJson(shallow(<CartDropdown  cartItems={mockCartItems}/>))).toMatchSnapshot();
})
