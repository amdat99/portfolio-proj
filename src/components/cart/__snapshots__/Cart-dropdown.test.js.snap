// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Greeting component 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
